compiler/rustc_typeck/src/check/method/mod.rs

```diff
@@ -468,6 +468,18 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         Some(InferOk { obligations, value: callee })
     }
 
+    /// Performs "universal function call" lookup. If lookup is successful, it will return the type
```

Comment:
Nit: I tend to this call this "fully qualified" lookup, but I guess the name hasn't caught on.

Reply:
My rational was to spell out the full name in the documentation to reduce the confusion that I hit with it (I've never seen the acronym "UFCs" in the context of Rust before!). However looking at [the reference](https://doc.rust-lang.org/reference/expressions/call-expr.html#disambiguating-function-calls), this term is no longer in use (for good reason I'd say!) so I think the best course of action is going to be to rename the function to match